From libc-ports-return-506-listarch-libc-ports=sources dot redhat dot com at sourceware dot org Thu Oct 19 20:15:34 2006
Return-Path: <libc-ports-return-506-listarch-libc-ports=sources dot redhat dot com at sourceware dot org>
Delivered-To: listarch-libc-ports at sources dot redhat dot com
Received: (qmail 19035 invoked by alias); 19 Oct 2006 20:15:33 -0000
Received: (qmail 19018 invoked by uid 22791); 19 Oct 2006 20:15:31 -0000
X-Spam-Status: No, hits=-1.3 required=5.0 	tests=AWL,BAYES_20,RCVD_ILLEGAL_IP,SPF_PASS
X-Spam-Check-By: sourceware.org
Received: from mail.codesourcery.com (HELO mail.codesourcery.com) (65.74.133.4)     by sourceware.org (qpsmtpd/0.31) with ESMTP; Thu, 19 Oct 2006 20:15:29 +0000
Received: (qmail 2094 invoked from network); 19 Oct 2006 20:15:28 -0000
Received: from unknown (HELO digraph.polyomino.org.uk) (joseph@127.0.0.2)   by mail.codesourcery.com with ESMTPA; 19 Oct 2006 20:15:28 -0000
Received: from jsm28 (helo=localhost) 	by digraph.polyomino.org.uk with local-esmtp (Exim 4.63) 	(envelope-from <joseph@codesourcery.com>) 	id 1GaeIY-0002Wg-TG 	for libc-ports@sourceware.org; Thu, 19 Oct 2006 20:15:26 +0000
Date: Thu, 19 Oct 2006 20:15:26 +0000 (UTC)
From: "Joseph S dot  Myers" <joseph at codesourcery dot com>
X-X-Sender: jsm28@digraph.polyomino.org.uk
To: libc-ports at sourceware dot org
Subject: Remove __i386__ conditional in ARM kernel-features.h
Message-ID: <Pine.LNX.4.64.0610192014350.8725@digraph.polyomino.org.uk>
MIME-Version: 1.0
Content-Type: TEXT/PLAIN; charset=US-ASCII
Mailing-List: contact libc-ports-help at sourceware dot org; run by ezmlm
Precedence: bulk
List-Subscribe: <mailto:libc-ports-subscribe at sourceware dot org>
List-Post: <mailto:libc-ports at sourceware dot org>
List-Help: <mailto:libc-ports-help at sourceware dot org>, <http://sourceware dot org/lists dot html#faqs>
Sender: libc-ports-owner at sourceware dot org
Delivered-To: mailing list libc-ports at sourceware dot org

The ARM kernel-features.h has an __i386__ conditional that is clearly
wrong.  (Before ports had their own kernel-features.h files, the
central one had (defined __i386__ || defined __arm__) here, so it
seems there was a mistake in copying into the ARM file.)

2006-10-19  Joseph S. Myers  <joseph@codesourcery.com>

	* sysdeps/unix/sysv/linux/arm/kernel-features.h: Remove __i386__
	conditional.

---
 sysdeps/unix/sysv/linux/arm/kernel-features.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: glibc-ports-2.5/sysdeps/unix/sysv/linux/arm/kernel-features.h
===================================================================
--- glibc-ports-2.5.orig/sysdeps/unix/sysv/linux/arm/kernel-features.h
+++ glibc-ports-2.5/sysdeps/unix/sysv/linux/arm/kernel-features.h
@@ -42,7 +42,7 @@
 #endif
 
 /* The vfork syscall on arm was definitely available in 2.4.  */
-#if __LINUX_KERNEL_VERSION >= 132097 && defined __i386__
+#if __LINUX_KERNEL_VERSION >= 132097
 # define __ASSUME_VFORK_SYSCALL		1
 #endif
 
